pkg/gadgets/audit-seccomp/gadget.go

```diff
@@ -106,9 +105,16 @@ func (t *Trace) Start(trace *gadgetv1alpha1.Trace) {
 	}
 
 	var err error
+
+	mountNsMap, err := t.resolver.TracerMountNsMap(traceName)
+	if err != nil {
+		trace.Status.OperationError = fmt.Sprintf("failed to create tracer: %s", err)
```

Comment:
```suggestion
		trace.Status.OperationError = fmt.Sprintf("failed to find trace: %s", err)
```

Reply:
Fixed it here and in the different places it was present.